css/feed.css

```diff
@@ -1,4 +1,6 @@
 .org-organization-admin-pages-entrypoint-card__card,
-.feed-identity-module__widgets {
+.feed-identity-module__widgets,
+.artdeco-toast-item--visible,
+.feed-new-update-pill__new-update-button {
```

Comment:
This is the "new posts" button that appears after you've been on scrolling in the feed for awhile.